requests/packages/urllib3/util/request.py

```diff
@@ -108,7 +108,7 @@ def rewind_body(body, body_pos):
         try:
             body_seek(body_pos)
         except (IOError, OSError):
-            raise UnrewindableBodyError("An error occured when rewinding request "
+            raise UnrewindableBodyError("An error occurred when rewinding request "
```

Comment:
Can you remove this patch please? urllib3 is a separate project that we vendor in, and we don't carry patches to it in tree. If you'd like to fix this, please make this patch upstream.